docs/modules/migration/partials/migration-guide-25.2.adoc

```diff
@@ -340,3 +340,49 @@ Check your custom CSS styles and adjust them if necessary.
 
 By default, the `SliderField` now shows an input field with the current value.
 To show only the read-only label again, set `valueEditable: false`.
+
+== SearchFormTableControl (Scout JS)
+
+The Scout JS `PageWithTable` will no longer filter its controls for the first `FormTableControl` holding a `Form` when looking for ist search filter but for the first `SearchFormTableControl`
+(see xref:howtos:scout-js/rest-service-how-to.adoc#page-with-table-search-form[Search Form] for more details).
+
+With this change existing `FormTableControl` will no longer be considered when looking for the search filter.
```

Comment:
Remove this sentence because it contains the same content as the first sentence.